### ISSUE TITLE
Players leave spleef premature and issues with reset.

### ISSUE THREAD

**Issue by thunerb** [alkarinv/BattleArena#146]
Players leave spleef during a game for some reason and after the game (on rejoining) they keep all the items from the spleef game (diamond shovel). My config is fairly similar to default with minor changes:
- Timers for match length slightly modified
- Reward at end modified slightly.

**Comment by alkarinv** [comment]
Can you tell me which plugins you have, maybe multiverse-inventories? this sounds like a plugin conflict.

**Comment by thunerb** [comment]
Yeah we have that. As we have almost 40 plugins, i'll just link you the page where we have a list of them:
http://ukgt.freeforums.org/full-plugin-list-t504.html

Would there be any way to resolve this short of removing the conflicting plugins?

**Comment by alkarinv** [comment]
There are always ways :) , so I'm thinking one of the multiverse ones, it might take a bit though to figure out.  Tonight I'm working on BattleShops, will try to get to BattleArena and this conflict
